src/pages/authorization/customer-accounts-api/index.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import fs from "fs";
+import path from "path";
+import PageRedoc from "../../../components/Page/PageRedoc";
+import useCategoryMeta from "../../../hooks/useCategoryMeta";
+
+const category = "authorization";
+const slug = "customer-accounts-api";
+
+const Page = ({ content }) => {
+  const subcategoryMeta = useCategoryMeta(category).items.filter(
+    (item) => item.slug === slug
+  )[0];
+
+  return (
+    <PageRedoc
+      name="customer-accounts-api"
+      content={content}
+      category={category}
+      desc={subcategoryMeta.desc}
+      title={subcategoryMeta.title}
+    />
+  );
+};
+
+export async function getStaticProps() {
+  const basePath = path.join(process.cwd(), "tmp");
+  const filePath = path.join(basePath, "redoc-customer-accounts-api.html");
+  const content = fs.readFileSync(filePath, "utf8");
```

Comment:
Reading from `tmp` directory and not handling any streams errors doesn't seem to me as a legit practice. Are you sure that this file will be always there? I'm not. 

<img width="1032" alt="Screenshot 2021-08-25 at 10 23 34" src="https://user-images.githubusercontent.com/28802251/130756133-415aaf05-5198-4f06-9ddb-72b3793ded48.png">